wsdlgo/encoder.go

```diff
@@ -508,7 +508,9 @@ func (p *{{.PortType}}) {{.Name}}( {{functionParamString .InParams}}) ({{functio
 		}
 	}{}
 	
-	if err = p.cli.RoundTrip(message, &out, "{{.SoapAction}}"); err != nil {
+	// simple context to pass SOAPAction--later versions to offer better call control
+	ctx := context.WithValue( context.Background(), "SOAPAction", "{{.SoapAction}}" )
```

Comment:
that space between the parenthesis and context.Background gives up that you haven't run gofmt in this file.. could you please do?